### ISSUE TITLE
Can't login after update to 2.18

### ISSUE THREAD

**Issue by SillyHats** [Qrome/marquee-scroller#211]
After updating from 2.16 to 2.18 using the web interface and the bin file below I can no longer log in. I've tried the username and password that was previously set and the default of admin and password.

marquee.ino.d1_mini_wide_2.18.bin

All other settings have persisted across the update. This same result happened after updating on two wide marquee scroller devices.

How can I log back in to the config page?

**Comment by Qrome** [comment]
You can view the password by running the arduino serial monitor with baud 115200 -- then press the reset button.  It will show in the log what the user / password is.  I have seen this happen if you tell chrome to save the password.

**Comment by SillyHats** [comment]
Thanks for the explanation about using the serial monitor. It looks like the username remained, but the password was changed to blank, which worked on the web interface.

I tried adding a password back to the bottom of the configure page. Clicking the save button appeared normal, but I couldn't get the password to save. After using the reset settings button on the side nav bar of the web interface I was able to add a password again.
